src/activities/hangman/hangman.js

```diff
@@ -200,6 +203,8 @@ function initSubLevel() {
     alreadyTypedLetters = new Array();
     currentWord = items.goodWord.translatedTxt;
     items.hidden.text = ""
+    items.guessedText.text=qsTr("Attempted: ");
```

Comment:
I would call form() here, and in form() instead of returning the string, I would directly update items.guessedText.text like stated below